toolchain/check/handle_class.cpp

```diff
@@ -200,12 +200,10 @@ static auto BuildClassDecl(Context& context, Parse::AnyClassDeclId node_id,
   if (introducer.extern_library.is_valid()) {
     context.TODO(node_id, "extern library");
   }
-  auto inheritance_kind =
-      introducer.modifier_set.HasAnyOf(KeywordModifierSet::Abstract)
-          ? SemIR::Class::Abstract
-      : introducer.modifier_set.HasAnyOf(KeywordModifierSet::Base)
-          ? SemIR::Class::Base
-          : SemIR::Class::Final;
+  SemIR::ClassFields::InheritanceKind inheritance_kind =
```

Comment:
```suggestion
  SemIR::Class::InheritanceKind inheritance_kind =
```

I think this would work, and feels a little more consistent with `SemIR::Class` below (which, I think the shorter bit helps)

Note, part of this will be that the `Class` versus `ClassFields` split is relatively new, but I generally don't think we should refer to the `FooFields` types directly (it's an abstraction working around C++, trying to hide C++ behaviors)